config/schedule.rb

```diff
@@ -1,9 +1,13 @@
 set :output, "#{ path }/log/daemons.log"
 
-every 1.hour do
-  rake 'joegattnet:one_hour'
-end
+# FIXME: We should be able to do this with stage
+if path == '/home/deployer/apps/joegattnet_v3'
+
+  every :hour do
+    rake '-s joegattnet:one_hour'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.